init.lua

```diff
@@ -3,42 +3,42 @@
 
 rules = {}
 
-
+local S = minetest.get_translator(minetest.get_current_modname())
 
 local items = {
-	"By playing on this server you agree to these rules:",
+	S("By playing on this server you agree to these rules:"),
 	"",
-	"1. No swearing, dating, or other inappriopriate behaviour.",
-	"2. No spamming, using all-caps, or any other method of chat abuse.",
-	"3. Don't be a cheater. No hacked clients.",
-	"4. Don't excessively spawnkill. Some spawn killing to stop players chasing",
-	"        the flag bearer is OK, but hogging the flag and repeatedly killing",
-	"        the same players definitely isn't.",
-	"5. Don't be a traitor. Don't:",
-	"    a. Dig blocks in your base to make it less secure or",
-	"       to trap team mates on purpose.",
-	"    b. Help the other team win.",
-	"    c. Change teams.",
-	"6. Don't leave the game whilst in a fight",
-	"7. Don't impersonate other community members",
-	"8. Do not share your password with ANYONE.",
-	"9. Avoid controversial topics like politics/religion",
-	"10. Moderator decisions are final.",
+	S("1. No swearing, dating, or other inappriopriate behaviour."),
+	S("2. No spamming, using all-caps, or any other method of chat abuse."),
+	S("3. Don't be a cheater. No hacked clients."),
+	S("4. Don't excessively spawnkill. Some spawn killing to stop players chasing"),
```

Comment:
breaking up the sentence like this, into multiple lines, is kinda awkward.

```lua
S("4. Don't excessively spawnkill."), " ",
S("Some spawn killing to stop players chasing the flag bearer is OK, but hogging the flag and repeatedly killing definitely isn't."),
```

Reply:
Newlines mess up the list 
![image](https://user-images.githubusercontent.com/28972859/93795675-90e5f800-fbee-11ea-947c-cd951e53e432.png)

Reply:
And translations can't handle it

Reply:
They don't even allow spaces around the = sign so I can't make the translation file more readable